src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -21,35 +21,35 @@ You can customize the creation of a new [BrowserContext](/javascript-api/xk6-bro
+| <BWIPT id="433"/> deviceScaleFactor | number  | `1`             | Sets the resolution ratio in physical pixels to the resolution in CSS pixels i.e. if set higher than `1`, then images will look sharper on high pixel density screens. See an [example](#devicescalefactor-example) below.                        |
+| extraHTTPHeaders                    | object  | `null`          | Contains additional HTTP headers to be sent with every request, where the keys are HTTP headers and values are HTTP header values.                                                                                                                |
+| <BWIPT id="435"/> geolocation       | object  | `null`          | Sets the user's geographical location. See [geolocation](#geolocation-options) for the options.                                                                                                                                                   |
+| <BWIPT id="436"/> hasTouch          | boolean | `false`         | Whether to simulate a device with touch events.                                                                                                                                                                                                   |
+| httpCredentials                     | object  | `null`          | Sets the credentials for HTTP authentication using Basic Auth. See: [httpCredentials](#httpcredentials-options).                                                                                                                                  |
+| ignoreHTTPSErrors                   | boolean | `false`         | Whether to ignore HTTPS errors that may be caused by invalid certificates.                                                                                                                                                                        |
+| isMobile                            | boolean | `false`         | Whether to simulate a mobile device.                                                                                                                                                                                                              |
+| javaScriptEnabled                   | boolean | `true`          | Whether to activate JavaScript support for the context.                                                                                                                                                                                           |
+| locale                              | string  | system          | Specifies the user's locale, such as `en-US`, `tr-TR`, etc.                                                                                                                                                                                       |
```

Comment:
I missed this in previous PRs, but we should quote string values, even if they're rendered as code.
```suggestion
| locale                              | string  | system          | Specifies the user's locale, such as `'en-US'`, `'tr-TR'`, etc.                                                                                                                                                                                       |
```